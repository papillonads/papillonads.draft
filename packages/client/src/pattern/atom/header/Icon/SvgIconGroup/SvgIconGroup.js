import React from 'react'
import { svgIconGroupShape } from '../SvgIcon.shape'
import { getSvgIconGroupPaths, getFill } from '../../../../../utility/asset'
import uuidv1 from 'uuid/v1'

const SvgIconGroup = ({ className, name, width, height, viewBox, fill, pathProps }) => {
  const paths = getSvgIconGroupPaths(name)

  if (!(paths && paths.length > 0)) {
    return null
  }

  return (
    <svg className={className} width={width} height={height} viewBox={viewBox}>
      <g fill={fill} fillRule="evenodd">
        {paths.map((path, index) => {
          return <path key={uuidv1()} d={paths[index]} fill={getFill(pathProps[index])} />
        })}
      </g>
    </svg>
  )
}

SvgIconGroup.defaultProps = {
  className: '',
}

SvgIconGroup.propTypes = svgIconGroupShape.isRequired

export default SvgIconGroup
