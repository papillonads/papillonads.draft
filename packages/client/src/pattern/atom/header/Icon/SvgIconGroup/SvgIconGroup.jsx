import React from 'react'
import uuidv1 from 'uuid/v1'
import { getSvgIconGroupPaths, getFill } from '../../../../../library/asset'
import { svgIconGroupDefaultProps, svgIconGroupPropTypes } from '../SvgIcon.props'

export const SvgIconGroup = ({ className, name, width, height, viewBox, fill, pathProps }) => {
  const paths = getSvgIconGroupPaths(name)

  if (!(paths && paths.length > 0)) {
    return null
  }

  return (
    <svg className={className} width={width} height={height} viewBox={viewBox}>
      <g fill={fill} fillRule="evenodd">
        {paths.map((path, index) => {
          return <path key={uuidv1()} d={paths[index]} fill={getFill(pathProps[index])} />
        })}
      </g>
    </svg>
  )
}

SvgIconGroup.defaultProps = svgIconGroupDefaultProps

SvgIconGroup.propTypes = svgIconGroupPropTypes
