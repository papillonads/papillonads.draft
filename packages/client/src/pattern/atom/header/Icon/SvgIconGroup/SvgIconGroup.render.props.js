export const logoRenderProps = {
  name: 'logo',
  width: 528,
  height: 96,
  viewBox: '0 0 528 96',
  pathProps: [{ fill: '#2D3C4D' }, { fill: '#EEA566' }, { fill: '#2D3C4D' }],
}

export const messagesRenderProps = {
  name: 'messages',
  width: 96,
  height: 96,
  viewBox: '0 0 96 96',
  fill: '#2D3C4D',
  pathProps: [],
}

export const notificationRenderProps = {
  name: 'notification',
  width: 96,
  height: 96,
  viewBox: '0 0 96 96',
  fill: '#2D3C4D',
  pathProps: [],
}

export const notificationGreyRenderProps = {
  name: 'notification-grey',
  width: 96,
  height: 96,
  viewBox: '0 0 96 96',
  fill: '#d4d1cd',
  pathProps: [],
}

export const profileRenderProps = {
  name: 'profile',
  width: 96,
  height: 96,
  viewBox: '0 0 96 96',
  fill: '',
  pathProps: [],
}

export const pinRenderProps = {
  name: 'pin',
  width: 96,
  height: 96,
  viewBox: '0 0 96 96',
  fill: '#2D3C4D',
  pathProps: [],
}

export const carsRenderProps = {
  name: 'autos',
  width: 96,
  height: 96,
  viewBox: '0 0 96 96',
  fill: '#2D3C4D',
  pathProps: [],
}

export const vacantiesRenderProps = {
  name: 'vacatures',
  width: 96,
  height: 96,
  viewBox: '0 0 96 96',
  fill: '#2D3C4D',
  pathProps: [],
}

export const offersRenderProps = {
  name: 'aanbiedingen',
  width: 96,
  height: 96,
  viewBox: '0 0 96 96',
  fill: '#2D3C4D',
  pathProps: [],
}

export const handshakeRenderProps = {
  name: 'handshake',
  width: 96,
  height: 96,
  viewBox: '0 0 96 96',
  fill: '#2D3C4D',
  pathProps: [],
}

export const arrowLargeRenderProps = {
  name: 'arrow-large',
  width: 96,
  height: 96,
  viewBox: '0 0 96 96',
  fill: '',
  pathProps: [],
}

export const categoryServicesRenderProps = {
  name: 'category-diensten',
  width: 96,
  height: 96,
  viewBox: '0 0 96 96',
  fill: '',
  pathProps: [
    { fill: '' },
    { fill: '#2D3C4D' },
    { fill: '#2D3C4D' },
    { fill: 'none' },
    { fill: '#2D3C4D' },
    { fill: '#2D3C4D' },
    { fill: '#2D3C4D' },
  ],
}

export const saveRenderProps = {
  name: 'save',
  width: 96,
  height: 96,
  viewBox: '0 0 96 96',
  fill: '#2D3C4D',
  pathProps: [],
}

export const saveGreenRenderProps = {
  name: 'save',
  width: 96,
  height: 96,
  viewBox: '0 0 96 96',
  fill: '#43aa8b',
  pathProps: [],
}

export const saveGreyRenderProps = {
  name: 'save',
  width: 96,
  height: 96,
  viewBox: '0 0 96 96',
  fill: '#818A95',
  pathProps: [],
}

export const resultsListRenderProps = {
  name: 'results-list',
  width: 96,
  height: 96,
  viewBox: '0 0 96 96',
  fill: '#2D3C4D',
  pathProps: [],
}
