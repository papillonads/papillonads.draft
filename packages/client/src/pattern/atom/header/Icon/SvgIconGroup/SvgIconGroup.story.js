import { storiesOf } from '@storybook/react'
import {
  logoRender,
  messagesRender,
  notificationRender,
  notificationGreyRender,
  profileRender,
  pinRender,
  carsRender,
  vacantiesRender,
  offersRender,
  handshakeRender,
  arrowLargeRender,
} from './SvgIconGroup.render'
import { getInfo } from '../../../../../../../../.storybook/utility'

const logoRenderInfo = getInfo('logo')
const messagesRenderInfo = getInfo('messages')
const notificationRenderInfo = getInfo('notification')
const notificationGreyRenderInfo = getInfo('notificationGrey')
const profileRenderInfo = getInfo('profile')
const pinRenderInfo = getInfo('pin')
const carsRenderInfo = getInfo('cars')
const vacantiesRenderInfo = getInfo('vacanties')
const offersRenderInfo = getInfo('offers')
const handshakeRenderInfo = getInfo('handshake')
const arrowLargeRenderInfo = getInfo('arrow-large')

storiesOf('Patttern/Atom/Header/Icon/SvgIconGroup', module)
  .add(logoRenderInfo.text, () => logoRender, logoRenderInfo.parameters)
  .add(messagesRenderInfo.text, () => messagesRender, messagesRenderInfo.parameters)
  .add(notificationRenderInfo.text, () => notificationRender, notificationRenderInfo.parameters)
  .add(notificationGreyRenderInfo.text, () => notificationGreyRender, notificationGreyRenderInfo.parameters)
  .add(profileRenderInfo.text, () => profileRender, profileRenderInfo.parameters)
  .add(pinRenderInfo.text, () => pinRender, pinRenderInfo.parameters)
  .add(carsRenderInfo.text, () => carsRender, carsRenderInfo.parameters)
  .add(vacantiesRenderInfo.text, () => vacantiesRender, vacantiesRenderInfo.parameters)
  .add(offersRenderInfo.text, () => offersRender, offersRenderInfo.parameters)
  .add(handshakeRenderInfo.text, () => handshakeRender, handshakeRenderInfo.parameters)
  .add(arrowLargeRenderInfo.text, () => arrowLargeRender, arrowLargeRenderInfo.parameters)
