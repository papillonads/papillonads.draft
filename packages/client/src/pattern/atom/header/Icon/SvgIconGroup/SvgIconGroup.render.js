/* eslint-disable react/jsx-props-no-spreading */
import React from 'react'
import SvgIconGroup from './SvgIconGroup'
import {
  logoRenderProps,
  messagesRenderProps,
  notificationRenderProps,
  notificationGreyRenderProps,
  profileRenderProps,
  pinRenderProps,
  carsRenderProps,
  vacantiesRenderProps,
  offersRenderProps,
  handshakeRenderProps,
  arrowLargeRenderProps,
  categoryServicesRenderProps,
  saveRenderProps,
  saveGreenRenderProps,
  saveGreyRenderProps,
  resultsListRenderProps,
  shoppingCartRenderProps,
  logoPostNlRenderProps,
} from './SvgIconGroup.render.props'

export const logoRender = <SvgIconGroup {...logoRenderProps} />
export const messagesRender = <SvgIconGroup {...messagesRenderProps} />
export const notificationRender = <SvgIconGroup {...notificationRenderProps} />
export const notificationGreyRender = <SvgIconGroup {...notificationGreyRenderProps} />
export const profileRender = <SvgIconGroup {...profileRenderProps} />
export const pinRender = <SvgIconGroup {...pinRenderProps} />
export const carsRender = <SvgIconGroup {...carsRenderProps} />
export const vacantiesRender = <SvgIconGroup {...vacantiesRenderProps} />
export const offersRender = <SvgIconGroup {...offersRenderProps} />
export const handshakeRender = <SvgIconGroup {...handshakeRenderProps} />
export const arrowLargeRender = <SvgIconGroup {...arrowLargeRenderProps} />
export const categoryServicesRender = <SvgIconGroup {...categoryServicesRenderProps} />
export const saveRender = <SvgIconGroup {...saveRenderProps} />
export const saveGreenRender = <SvgIconGroup {...saveGreenRenderProps} />
export const saveGreyRender = <SvgIconGroup {...saveGreyRenderProps} />
export const resultsListRender = <SvgIconGroup {...resultsListRenderProps} />
export const shoppingCartRender = <SvgIconGroup {...shoppingCartRenderProps} />
export const logoPostNlRender = <SvgIconGroup {...logoPostNlRenderProps} />
/* eslint-enable react/jsx-props-no-spreading */
