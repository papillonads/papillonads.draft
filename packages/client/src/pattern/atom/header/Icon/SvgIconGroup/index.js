import SvgIconGroup from './SvgIconGroup'

export default SvgIconGroup
