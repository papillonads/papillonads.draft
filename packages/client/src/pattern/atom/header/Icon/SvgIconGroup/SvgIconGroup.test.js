import { logoRender, messagesRender, notificationRender, notificationGreyRender, profileRender, pinRender } from './SvgIconGroup.render'

describe('<SvgIcon />', () => {
  describe('Snaphot', () => {
    test('must match logoRender', () => {
      expect(global.renderToJSON(logoRender)).toMatchSnapshot()
    })

    test('must match messagesRender', () => {
      expect(global.renderToJSON(messagesRender)).toMatchSnapshot()
    })

    test('must match notificationRender', () => {
      expect(global.renderToJSON(notificationRender)).toMatchSnapshot()
    })

    test('must match notificationGreyRender', () => {
      expect(global.renderToJSON(notificationGreyRender)).toMatchSnapshot()
    })

    test('must match profileRender', () => {
      expect(global.renderToJSON(profileRender)).toMatchSnapshot()
    })

    test('must match pinRender', () => {
      expect(global.renderToJSON(pinRender)).toMatchSnapshot()
    })
  })
})
