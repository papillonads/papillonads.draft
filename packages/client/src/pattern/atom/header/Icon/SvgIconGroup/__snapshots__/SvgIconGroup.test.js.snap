// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SvgIcon /> Snaphot must match logoRender 1`] = `
<svg
  className=""
  height="96"
  viewBox="0 0 528 96"
  width="528"
>
  <g
    fillRule="evenodd"
  >
    <path
      d="M119.692 23.506v6.355l3.857.219-2.328 33.089c-.219 1.826-.583 2.337-1.965 2.556l-2.256.22v6.135h17.319v-6.355l-3.639-.219 1.456-26.296h.145l10.26 27.684h7.641l10.552-28.049h.218l1.237 24.397c.073 1.826-.218 2.264-1.601 2.483l-2.256.22v6.135h17.392v-6.355l-3.711-.219-2.256-33.235c-.219-1.534.51-2.191 1.892-2.41l2.183-.22v-6.135H157.75l-11.133 30.897h-.219l-10.697-30.897h-16.009zm92.488 13.221c-5.53-2.118-10.842-2.63-13.389-2.63-13.17 0-19.574 8.254-19.574 21.11 0 9.934 3.784 17.823 14.553 17.823 4.657 0 7.932-1.973 9.678-4.675l.291.073.291 3.652h11.789v-5.843l-2.038-.147c-1.237-.073-1.6-.584-1.6-2.045V36.727zM202.14 59.955c0 2.922-2.11 5.624-6.113 5.624-5.24 0-6.55-5.259-6.55-10.883 0-6.94 1.82-13.075 8.15-13.075 1.82 0 3.567.365 4.513.73v17.604zm37.111-17.677c.8 0 1.383.073 1.82.22.509.219.654.584.654 1.752v4.821l8.223-.584V35.339c-1.965-.876-4.584-1.242-6.695-1.242-6.04 0-9.168 3.653-10.333 6.574h-.145v-5.624h-13.317v6.136l2.474.219c1.237.073 1.674.584 1.674 2.045V63.68c0 1.68-.218 2.191-1.746 2.41l-2.329.22v5.77h20.157v-6.063l-6.04-.292V49.071c0-3.871 2.11-6.793 5.603-6.793zm28.307 14.39c1.528-.22 2.91-.438 4.366-1.023l6.986 16.435h12.225v-6.063l-3.93-.219-6.622-14.316c3.493-2.849 5.458-6.501 5.967-10.007l3.275-.438v-5.99h-16.082v5.916l1.892.22c.946.219 1.528.438 1.528 1.387 0 3.36-5.166 6.501-9.605 7.086V20h-14.263v6.136l2.475.219c1.382.073 1.673.584 1.673 2.118v35.353c0 1.68-.436 2.045-1.673 2.264l-2.33.22v5.77h17.392v-6.063l-3.274-.219v-9.13zm25.032-14.536h4.948v17.896c0 6.209.582 13.075 11.934 13.075 3.784 0 7.277-.95 9.315-2.338l-1.674-6.939c-1.164.657-2.91 1.242-4.876 1.242-2.91 0-4.438-1.169-4.438-5.478V42.132h8.659l.364-7.085h-9.023v-8.4h-5.604c-1.528 4.383-3.492 8.254-8.732 9.203l-.873 6.282zM325.771 78.8c0 1.826-.364 2.118-1.746 2.337l-2.256.293v5.77h20.375v-6.063l-6.258-.365v-9.495c1.965 1.095 4.657 1.826 7.277 1.826 10.26 0 15.572-7.89 15.572-20.672 0-11.76-4.584-18.334-13.098-18.334-4.075 0-8.15 1.461-10.406 5.479h-.145v-4.529h-13.463v6.136l2.402.146c1.164.073 1.746.73 1.746 2.118V78.8zm10.188-31.263c0-2.337 1.892-5.697 6.112-5.697 5.312 0 6.404 5.843 6.404 11.833 0 6.647-1.674 11.833-7.131 11.833-1.82 0-3.93-.657-5.385-1.753V47.537zM376.854 20h-14.262v6.136l2.401.219c1.383.073 1.747.584 1.747 2.118V63.68c0 1.826-.291 2.118-1.674 2.337l-2.329.293v5.77h17.901v-6.063l-3.784-.219V20zm40.532 16.727c-5.53-2.118-10.842-2.63-13.39-2.63-13.17 0-19.574 8.254-19.574 21.11 0 9.934 3.784 17.823 14.554 17.823 4.657 0 7.932-1.973 9.678-4.675l.291.073.291 3.652h11.789v-5.843l-2.038-.147c-1.237-.073-1.6-.584-1.6-2.045V36.727zm-10.042 23.228c0 2.922-2.11 5.624-6.112 5.624-5.24 0-6.55-5.259-6.55-10.883 0-6.94 1.82-13.075 8.15-13.075 1.82 0 3.566.365 4.512.73v17.604zm50.792-23.228c-5.53-2.118-10.842-2.63-13.39-2.63-13.17 0-19.574 8.254-19.574 21.11 0 9.934 3.784 17.823 14.554 17.823 4.657 0 7.932-1.973 9.678-4.675l.291.073.291 3.652h11.789v-5.843l-2.038-.147c-1.237-.073-1.6-.584-1.6-2.045V36.727zm-10.042 23.228c0 2.922-2.11 5.624-6.112 5.624-5.24 0-6.55-5.259-6.55-10.883 0-6.94 1.82-13.075 8.15-13.075 1.82 0 3.566.365 4.512.73v17.604zm16.227-17.823h4.949v17.896c0 6.209.582 13.075 11.934 13.075 3.784 0 7.276-.95 9.314-2.338l-1.674-6.939c-1.164.657-2.91 1.242-4.875 1.242-2.91 0-4.439-1.169-4.439-5.478V42.132h8.66l.363-7.085h-9.023v-8.4h-5.603c-1.528 4.383-3.493 8.254-8.732 9.203l-.874 6.282zm49.192-.657c.873.292 1.018.803 1.018 1.972v2.995l7.204-.585v-9.13c-3.201-1.753-7.567-2.63-11.57-2.63-8.732 0-15.208 4.164-15.208 11.98 0 7.888 5.53 9.568 10.77 10.956 4.074.95 6.912 1.68 6.912 4.82 0 3.142-2.255 4.237-5.675 4.237-1.456 0-2.984-.292-3.93-.657-.873-.292-1.091-.73-1.091-1.826V60.32l-7.277.584v9.715c2.838 1.534 8.44 2.484 12.88 2.484 9.387 0 15.354-4.456 15.354-12.345 0-7.596-5.749-9.276-11.061-10.883-3.857-1.169-6.986-1.753-6.986-5.04 0-2.557 2.183-3.798 5.094-3.798 1.164 0 2.474.073 3.566.438z"
      fill="#2D3C4D"
    />
    <path
      d="M48.018 0C21.498 0 0 21.492 0 48s21.497 48 48.018 48 48.019-21.492 48.019-48S74.539 0 48.018 0"
      fill="#EEA566"
    />
    <path
      d="M64.19 36.65a8.877 8.877 0 0 1 0 12.782L51.196 62.09c-3.623 3.533-9.497 3.533-13.12 0l19.554-19.05L44.64 30.38a8.874 8.874 0 0 1 0-12.78l19.55 19.05zM57.21 77.12H29.771c0-4.995 4.12-9.047 9.206-9.047H66.417c0 4.996-4.121 9.047-9.206 9.047"
      fill="#2D3C4D"
    />
  </g>
</svg>
`;

exports[`<SvgIcon /> Snaphot must match messagesRender 1`] = `
<svg
  className=""
  height="96"
  viewBox="0 0 96 96"
  width="96"
>
  <g
    fill="#2D3C4D"
    fillRule="evenodd"
  >
    <path
      d="M73.926 66.961c1.626-1.552 5.789-6.066 6.05-11.825.173-3.808-1.359-7.357-4.552-10.551a2 2 0 1 0-2.828 2.828c2.36 2.36 3.499 4.891 3.385 7.522-.226 5.242-5.205 9.488-5.251 9.527a2.007 2.007 0 0 0-.689 1.187c-.033.184-.536 3.217.874 7.298a356.284 356.284 0 0 1-7.86-4.654 2.01 2.01 0 0 0-1.157-.292c-.116.005-11.819.556-16.153-6.999a2 2 0 0 0-3.469 1.991c4.954 8.632 16.332 9.052 19.2 9.021 1.994 1.212 9.234 5.578 13.643 7.774a2 2 0 0 0 2.45-3.041c-3.485-4.338-3.706-8.303-3.643-9.786z"
      fill=""
    />
    <path
      d="M70.01 36.032C70.01 24.986 57.898 16 43.01 16s-27 8.986-27 20.032c0 5.922 3.475 11.5 9.534 15.302 1.664 1.044 1.146 4.524-1.42 9.548-.518 1.015-.145 1.827.04 2.129.363.595 1.086.989 1.781.989h.002c.958 0 1.783-.493 5.971-3.25 2.403-1.582 6.838-4.502 8.101-4.807.987.08 1.992.121 2.99.121 14.889 0 27.001-8.986 27.001-20.032zM39.978 51.937c-1.733 0-4.543 1.713-10.068 5.347 1.287-4.461.541-7.592-2.239-9.337-4.869-3.056-7.661-7.398-7.661-11.915 0-8.84 10.318-16.032 23-16.032s23 7.192 23 16.032-10.318 16.032-23 16.032c-.933 0-1.872-.04-2.792-.118a4.722 4.722 0 0 0-.24-.009z"
      fill=""
    />
  </g>
</svg>
`;

exports[`<SvgIcon /> Snaphot must match notificationGreyRender 1`] = `
<svg
  className=""
  height="96"
  viewBox="0 0 96 96"
  width="96"
>
  <g
    fill="#d4d1cd"
    fillRule="evenodd"
  >
    <path
      d="M54.693 72.066a1.988 1.988 0 0 0-2.651.944c-.819 1.726-2.585 2.845-4.496 2.845s-3.675-1.119-4.497-2.845a1.986 1.986 0 0 0-2.649-.944 1.991 1.991 0 0 0-.944 2.652 9.002 9.002 0 0 0 8.091 5.115 9.005 9.005 0 0 0 8.091-5.115 1.992 1.992 0 0 0-.945-2.652M47.221 20.145a2.987 2.987 0 0 1 2.984 2.984 2.986 2.986 0 0 1-2.984 2.984 2.988 2.988 0 0 1-2.984-2.984 2.988 2.988 0 0 1 2.984-2.984m23.246 42.61a2.386 2.386 0 0 0-.192-.276c-.047-.053-4.494-5.584-4.494-18.458 0-8.996-4.902-13.433-9.012-15.57a1.99 1.99 0 1 0-1.836 3.529c4.559 2.37 6.866 6.422 6.866 12.041 0 13.355 4.43 19.735 5.299 20.857.42.767.431 1.291.303 1.51-.134.238-.644.516-1.629.516h-35.8c-.951 0-1.442-.257-1.572-.478-.144-.242-.108-.825.283-1.598.921-1.206 5.264-7.568 5.264-20.807 0-13.548 13.787-13.927 13.926-13.927.249 0 .484-.051.704-.134 3.191-.632 5.61-3.454 5.61-6.83 0-3.84-3.124-6.962-6.965-6.962a6.97 6.97 0 0 0-6.962 6.962c0 1.552.514 2.98 1.375 4.139-5.467 1.909-11.668 6.578-11.668 16.752 0 12.872-4.437 18.382-4.472 18.423a2 2 0 0 0-.237.344c-1.122 2.072-1.22 4.089-.277 5.675.93 1.559 2.702 2.416 4.992 2.416h35.799c2.311 0 4.164-.915 5.08-2.51.912-1.589.771-3.635-.385-5.614"
      fill=""
    />
  </g>
</svg>
`;

exports[`<SvgIcon /> Snaphot must match notificationRender 1`] = `
<svg
  className=""
  height="96"
  viewBox="0 0 96 96"
  width="96"
>
  <g
    fill="#2D3C4D"
    fillRule="evenodd"
  >
    <path
      d="M54.693 72.066a1.988 1.988 0 0 0-2.651.944c-.819 1.726-2.585 2.845-4.496 2.845s-3.675-1.119-4.497-2.845a1.986 1.986 0 0 0-2.649-.944 1.991 1.991 0 0 0-.944 2.652 9.002 9.002 0 0 0 8.091 5.115 9.005 9.005 0 0 0 8.091-5.115 1.992 1.992 0 0 0-.945-2.652M47.221 20.145a2.987 2.987 0 0 1 2.984 2.984 2.986 2.986 0 0 1-2.984 2.984 2.988 2.988 0 0 1-2.984-2.984 2.988 2.988 0 0 1 2.984-2.984m23.246 42.61a2.386 2.386 0 0 0-.192-.276c-.047-.053-4.494-5.584-4.494-18.458 0-8.996-4.902-13.433-9.012-15.57a1.99 1.99 0 1 0-1.836 3.529c4.559 2.37 6.866 6.422 6.866 12.041 0 13.355 4.43 19.735 5.299 20.857.42.767.431 1.291.303 1.51-.134.238-.644.516-1.629.516h-35.8c-.951 0-1.442-.257-1.572-.478-.144-.242-.108-.825.283-1.598.921-1.206 5.264-7.568 5.264-20.807 0-13.548 13.787-13.927 13.926-13.927.249 0 .484-.051.704-.134 3.191-.632 5.61-3.454 5.61-6.83 0-3.84-3.124-6.962-6.965-6.962a6.97 6.97 0 0 0-6.962 6.962c0 1.552.514 2.98 1.375 4.139-5.467 1.909-11.668 6.578-11.668 16.752 0 12.872-4.437 18.382-4.472 18.423a2 2 0 0 0-.237.344c-1.122 2.072-1.22 4.089-.277 5.675.93 1.559 2.702 2.416 4.992 2.416h35.799c2.311 0 4.164-.915 5.08-2.51.912-1.589.771-3.635-.385-5.614"
      fill=""
    />
  </g>
</svg>
`;

exports[`<SvgIcon /> Snaphot must match pinRender 1`] = `
<svg
  className=""
  height="96"
  viewBox="0 0 96 96"
  width="96"
>
  <g
    fill="#2D3C4D"
    fillRule="evenodd"
  >
    <path
      d="M78.426 80.425a1.992 1.992 0 0 1-1.414-.586l-18-18a2 2 0 1 1 2.828-2.828l18 18a2 2 0 0 1-1.414 3.414zM41.404 72a2 2 0 0 1-1.414-.586c-5.973-5.973-3.874-12.278-1.11-15.859L25.133 35.462c-2.118.575-5.856.942-8.973-2.176a2 2 0 0 1 0-2.828L30.458 16.16a2 2 0 0 1 2.828 0c3.117 3.117 2.752 6.854 2.176 8.973L55.554 38.88c1.258-.978 3.188-2.145 5.609-2.572 2.548-.451 6.393-.176 10.251 3.683a2 2 0 0 1 0 2.828L42.818 71.414a1.999 1.999 0 0 1-1.414.586zm1.392-14.86c-1.234 1.242-4.53 5.255-1.274 9.915l25.532-25.532c-1.666-1.162-3.404-1.59-5.195-1.276-2.444.432-4.248 2.078-4.742 2.572-.68.68-1.747.781-2.543.236l-22.639-15.49a2 2 0 0 1-.66-2.545c.05-.103 1.04-2.207.288-4.308l-10.85 10.85c2.102.752 4.207-.239 4.32-.293a2.013 2.013 0 0 1 2.532.666l14.129 20.649 6.777-6.777 2.828 2.828-8.453 8.453-.05.052z"
      fill=""
    />
  </g>
</svg>
`;

exports[`<SvgIcon /> Snaphot must match profileRender 1`] = `
<svg
  className=""
  height="96"
  viewBox="0 0 96 96"
  width="96"
>
  <g
    fill=""
    fillRule="evenodd"
  >
    <path
      d="M78 78c0-20-16-18-16-18L48 74 34 60s-16-2-16 18"
      fill=""
    />
  </g>
</svg>
`;
